models/pancakeswap/bnb/pancakeswap_v2_bnb_amm_trades.sql

```diff
@@ -21,22 +21,22 @@ WITH dexs AS
     SELECT
         t.evt_block_time                                                                AS block_time,
         t.to                                                                            AS taker,
-        ''                                                                              AS maker,
-        CASE WHEN amount0Out = '0' THEN amount1Out ELSE amount0Out END                  AS token_bought_amount_raw,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN amount1In ELSE amount0In END AS token_sold_amount_raw,
-        cast(NULL as double)                                                            AS amount_usd,
-        CASE WHEN amount0Out = '0' THEN f.token1 ELSE f.token0 END                      AS token_bought_address,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN f.token1 ELSE f.token0 END   AS token_sold_address,
+        CAST(NULL AS VARBINARY)                                                         AS maker,
+        CASE WHEN amount1Out = UINT256 '0' THEN amount1Out ELSE amount0Out END          AS token_bought_amount_raw,
```

Comment:
Is `amount10Out` correct here? The current version uses `amount0Out`. Same on line 28

Reply:
Same in other files

Reply:
Should be `amount0Out` as `pancakeswap_v2_bnb.PancakePair_evt_Swap` does not have `amount10Out`